tests/spectron/spellChecker.spectron.js

```diff
@@ -114,7 +114,7 @@ describe('Tests for spellChecker', () => {
         return app.client
             .windowByIndex(0)
             .getValue('#tag').then((value) => {
-                expect(value).toBe('coming ');
+                expect(value !== 'comming').toBeTruthy();
```

Comment:
why are we doing a negative check here?

Reply:
This is required as spellchecker context menu suggestion is not consistent, so we need to make sure the misspelt `comming` is changed to some correct spelt word 😄.